theano/configdefaults.py

```diff
@@ -130,7 +130,7 @@ def __str__(self):
     'conv.assert_shape',
     "If False, AbstractConv* ops won't add assert that verify that"
     " the user provided shapes are also the one at run time",
-    BoolParam(True),
+    BoolParam(False),
```

Comment:
Maybe also update the documentation to something like "If True, AbstractConv* ops will verify that user-provided compile time shapes match the runtime shapes (debugging option, may slow down compilation)".